controller/users_blackbox_test.go

```diff
@@ -78,7 +78,15 @@ func (s *TestUsersSuite) TestUpdateUserOK() {
 	newBio := "new bio"
 	newProfileURL := "http://new.profile.url/url"
 	secureService, secureController := s.SecuredController(identity)
-	updateUsersPayload := createUpdateUsersPayload(&newEmail, &newFullName, &newBio, &newImageURL, &newProfileURL)
+
+	contextInformation := map[string]interface{}{
```

Comment:
Internally I'm using `workitem.Fields`.
At the service interface it is being defined https://github.com/almighty/almighty-core/pull/1007/files#diff-5a990e44c9a8391f2956c4b6d0128401R171  as an empty interface 

( the same way we are doing for wi and wit )